src/library/indicators.jl

```diff
@@ -16,3 +16,49 @@ function ar1_whitenoise(x::AbstractVector{T}) where {T<:Real}
     n = length(x)
     return (view(x, 2:n)' * view(x, 1:n-1)) / (view(x, 1:n-1)' * view(x, 1:n-1))
 end
+
+"""
+    LowfreqPowerSpectrum(x::AbstractVector)
+
+Returns a number between `0` and `1` that characterizes the amount of power contained
+in the low frequencies of the power density spectrum of `x`.
+A result of `0.1` means that 10% of the integrated power is contained in low frequencies.
+
+The computation is based on `q_lofreq::Real`, a number between `0` and `1` that
+determines which frequencies are considered to be "low".
+"""
+Base.@kwdef struct LowfreqPowerSpectrum <: PrecomputableFunction
+    q_lofreq::Real = 0.1
```

Comment:
So to be clear the units of this is frequency units, right? And the total amount of possible frequencies in the signal change, increase, with increasing signal length. Could we instead make `q_lofreq` a percentage? 0.1 means the 10% lowest frequencies. I think that is more usable as it doesn't depend on the length of input timeseries.